doc/users/whats_new/updated_backend_wx.rst

```diff
@@ -0,0 +1,10 @@
+wx backend has been updated
+---------------------------
+The wx backend can now be used with both wxPython classic and
+`Phoenix <http://wxpython.org/Phoenix/docs/html/main.html>`__.
+
+wxPython classic has to be at least version 2.8.12 and works on Python 2.x,
+wxPython Phoenix needs a current snapshot and works on Python 2.7 and 3.4+.
+
+User code is responsible to set the wxPython version you want to use, see for
+example the `examples\user_interfaces\embedding_in_wx2.py`.
```

Comment:
"to set" --> "for setting". I would also make it clear that the user
doesn't have to set anything if they want just the default. The language
here could spook some people into thinking that they may have to specify
this, and/or that they have to update existing code.

On Tue, Apr 28, 2015 at 10:56 AM, Werner F Bruhin <notifications@github.com>
wrote:

> In doc/users/whats_new/updated_backend_wx.rst
> <https://github.com/matplotlib/matplotlib/pull/3421#discussion_r29250782>:
>
> > @@ -0,0 +1,10 @@
> > +wx backend has been updated
> > +---------------------------
> > +The wx backend can now be used with both wxPython classic and
> > +`Phoenix <http://wxpython.org/Phoenix/docs/html/main.html>`__.
> > +
> > +wxPython classic has to be at least version 2.8.12 and works on Python 2.x,
> > +wxPython Phoenix needs a current snapshot and works on Python 2.7 and 3.4+.
> > +
> > +User code is responsible to set the wxPython version you want to use, see for
> > +example the `examples\user_interfaces\embedding_in_wx2.py`.
>
> Hhm, the comment explains the two methods I know on how to select a
> specific version of wxPython. Either set the version in wx.pth in
> site-packages or uncomment the line two lines to use wxversion.
>
> Would this be better?
>
> "User code is responsible to set the wxPython version you want to use, how
> to do this
> is explained at the beginning of
> examples\user_interfaces\embedding_in_wx2.py."
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/matplotlib/matplotlib/pull/3421/files#r29250782>.
>